### DISCUSSION TITLE
BL616 USB WIFI

### DISCUSSION THREAD

**Discussion by maaamcube** [cherry-embedded/CherryUSB#263] [Q&A] [answered]
Hi @sakumisu,
I want to know if the bl616 usb wifi is made to be put in an esp32 and connect a bl616 usb device to it and send the bl616 usb device through wifi ?
Thanks.
PS: Where can i contact you ? because, i tag you in the discord server and you are not there always.

**Comment by sakumisu** [comment]
bl616 usb wifi only supports high speed for performance.

**Reply by maaamcube** [reply]
so it made to send the bl616 usb device through wifi using esp32 for example ?

**Reply by sakumisu** [reply, marked as answer]
> so it made to send the bl616 usb device through wifi using esp32 for example ?

yes

**Reply by maaamcube** [reply]
Thanks, where can i contact you for quick answering between ?

**Reply by sakumisu** [reply]
> Thanks, where can i contact you for quick answering between ?

In discord i think.
